tests/applications.js

```diff
@@ -0,0 +1,54 @@
+import test from 'ava';
+import _ from 'lodash';
+
+import applications from '../lib/applications';
+import merged from './fixtures/applications/objects/model-merged.js';
+
+const reqObj = {
```

Comment:
@Snugug This is a leftover from me splitting out my massive, massive PR. I will be using this object extensively in the routes testing that is coming up. Within that testing `req` is commonly used.